Dockerfile

```diff
@@ -17,43 +17,53 @@ RUN apt update && \
 RUN yes | unminimize
 
 
-# Install Ubuntu packages
+# Suggested build environment for Ruby and Python, per pyenv, even though we're building ourselves
```

Comment:
These are the packages that building Python assumes, no, not Ruby per se? Even though Ruby presumably needs `make` and `build-essential`?